clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -6418,6 +6418,12 @@ def warn_signed_bitfield_enum_conversion : Warning<
   InGroup<BitFieldEnumConversion>, DefaultIgnore;
 def note_change_bitfield_sign : Note<
   "consider making the bitfield type %select{unsigned|signed}0">;
+def warn_ms_bitfield_mismatched_storage_packing : Warning<
+  "bit-field %0 of type %1 has a different storage size (%2 vs %3 bytes) than the "
+  "preceding bit-field and may not be packed under MSVC ABI">,
```

Comment:
```suggestion
  "bit-field %0 of type %1 has a different storage size than the "
  "preceding bit-field (%2 vs %3 bytes) and will not be packed under "
  "the MSVC ABI">,
```
I think it’s a bit clearer this way.

Reply:
Thanks, I'm terrible at wording (I prodded discord specifically to get diagnostic wording)